src/app/components/main-page/main-page.component.scss

```diff
@@ -0,0 +1,58 @@
+@import '../../styles/colors';
+
+.wrapper {
+  display: flex;
+  justify-content: space-around;
+  flex-direction: column;
+  min-width: 300px;
+
+  .container {
+    flex: 1 1 auto;
+    padding: 20px;
+    margin: 0;
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+    background: #fafafa;
+    width: 80vw;
+    align-self: center;
+    height: 90vh;
+    min-height: fit-content;
+
+    .product-card {
+      background: #f2f2f2;
+      margin: 20px;
+      height: 40%;
+      width: 25%;
+      min-height: fit-content;
+      min-width: 200px;
+      box-shadow: 0 0 40px -10px rgba(0,0,0,0.75);
+    }
+  }
+
+  .header {
+    flex-shrink: 0;
+    padding: 20px;
+    background: black;
+    color: $white;
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    min-width: 300px;
+  }
+
+  .footer {
+    display: flex;
+    align-content: center;
+    order: 1;
+    width: 100vw;
+    padding: 20px;
+    background: black;
+    color: $white;
+    margin: 0;
+  }
+}
+
+* {
```

Comment:
it very very very common style. why is it living in one of component styles?

Reply:
done